src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -349,12 +349,7 @@ public void TestExitTime()
                     break;
             }
 
-            Assert.True(p.ExitTime.ToUniversalTime() >= timeBeforeProcessStart,
-                $@"TestExitTime is incorrect. " +
-                $@"TimeBeforeStart {timeBeforeProcessStart} Ticks={timeBeforeProcessStart.Ticks}, " +
-                $@"ExitTime={p.ExitTime}, Ticks={p.ExitTime.Ticks}, " +
-                $@"ExitTimeUniversal {p.ExitTime.ToUniversalTime()} Ticks={p.ExitTime.ToUniversalTime().Ticks}, " +
-                $@"NowUniversal {DateTime.Now.ToUniversalTime()} Ticks={DateTime.Now.Ticks}");
+            Assert.InRange(p.ExitTime.ToUniversalTime(), timeBeforeProcessStart, DateTime.MaxValue);
```

Comment:
perhaps `DateTime.MaxValue` => `timeBeforeProcessStart.AddHours(1)`

Reply:
I only wanted to change the Assert.Trues to use Assert.InRange.  I don't want to add additional bounds in this PR that weren't there before.